rules/eslint/best-practices/eslint.js

```diff
@@ -9,94 +9,100 @@ module.exports = {
     // specify the maximum cyclomatic complexity allowed in a program
     "complexity": [0, 11],
     // require return statements to either always or never specify values
-    "consistent-return": 2,
+    "consistent-return": 0,
     // specify curly brace conventions for all control statements
-    "curly": [2, "all"],
+    "curly": [0, "all"],
     // require default case in switch statements
     "default-case": 0,
     // encourages use of dot notation whenever possible
-    "dot-notation": [2, { "allowKeywords": true }],
+    "dot-notation": [0, { "allowKeywords": true }],
     // enforces consistent newlines before or after dots
     "dot-location": 0,
     // require the use of === and !==
-    "eqeqeq": 2,
+    "eqeqeq": 0,
     // make sure for-in loops have an if statement
     "guard-for-in": 0,
     // disallow the use of alert, confirm, and prompt
-    "no-alert": 2,
+    "no-alert": 0,
     // disallow use of arguments.caller or arguments.callee
-    "no-caller": 2,
+    "no-caller": 0,
```

Comment:
Maybe I'm missing something - same question here, tho.  These properties are deprecated and explicitly disallowed in strict mode.